### ISSUE TITLE
Possible Item Duplication

### ISSUE THREAD

**Issue by TheBananaofBlue** [ValkyrienSkies/Eureka#270]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [ ] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.19

### Mod Loader

Forge

### Issue description

We had a storage lecturn from ars nouveau on the ship when it was assembled. It lost all its chest bindings and items started to spawn from our storage but our items were still in the chests. Was fixed by killing the item entities. 

### Issue reproduction

Have ars nouveau and make a storage system with the lecturn then assemble a ship

**Comment by PriestOfFern** [comment]
Known issue, VS2 checks for block entities that interface clearables, and manually clears them if they are.

This however means  that any blocks that use custom classes for containers don’t get cleared. 

In any case, it would be much easier to ask mods devs to use the clearable class, rather than have VS2 manually check for every single container in MC modding history that used custom classes

**Comment by MCAlexisYT** [comment]
@PriestOfFerns Yeah, that would probably be hundreds of kilobytes worth of data at most.

**Comment by Creeperz653** [comment]
beds duplicate

**Comment by narnianpony** [comment]
+1 to bed duplication
